server/pxf-hive/src/main/java/org/greenplum/pxf/plugins/hive/HiveAccessor.java

```diff
@@ -123,8 +209,29 @@ public HiveAccessor() {
      */
     @Override
     public void initialize(RequestContext context) {
+
         super.initialize(context);
-        HiveMetadata metadata;
+
+        // determine if predicate pushdown is allowed by configuration
+        if (configuration.get(PXF_PPD_HIVE, "true").equalsIgnoreCase("true")) {
+            isPredicatePushdownAllowed = true;
```

Comment:
Should we explicitly set the flag to prevent any second guessing here?
```suggestion
        isPredicatePushdownAllowed = configuration.get(PXF_PPD_HIVE, "true").equalsIgnoreCase("true");
        if (isPredicatePushdownAllowed) {
```